docs/parameters.rst

```diff
@@ -129,6 +129,9 @@ integers::
                 elif value[:1] == '0':
                     return int(value, 8)
                 return int(value, 10)
+        except TypeError:
+            self.fail('expected string for int() conversion, got %s of type %s'
+                      % (value, type(value).__name__), param, ctx)
```

Comment:
GitHub is making it look like the added lines need one more level of indentation to match the level of the `except ValueError` below.

Reply:
Fixed in 0049354040156cbbc521f0e6a62d32ebab780958